src/bika/coa/reportview.py

```diff
@@ -845,7 +846,24 @@ def get_formatted_uncertainty(self, analysis):
         if uncertainty:
             return "&plusmn; {}".format(uncertainty)
         return
-    
+
+    def get_formatted_specs_hydro(self, model, analysis):
+        specs = analysis.getResultsRange()
+        fs = ''
+        precision = analysis.getPrecision()
+        precision_formatting = "{:." + str(precision) + "f}"
+        if specs.get('min', None) and specs.get('max', None):
+            min_val = precision_formatting.format(float(specs['min']))
```

Comment:
`min_val = round(specs['min'], precision)`

Reply:
Gave it a go and, round doesn't preserve the trailling zeroes. So 
round(5,449, 5) = 5,449
whilst 
precision = 5
precision_formatting.format(float(5,449)) = 5.44900
![image](https://github.com/bikalims/bika.coa/assets/104898641/042ef45d-7f69-453d-932f-d89e1bda9d47)